### ISSUE TITLE
unminify bundle.js

### ISSUE THREAD

**Issue by rezamukti** [klanjabrik/AlBarokah#5]
Terima kasih sudah berbagi pos ini Mas,
kalau bisa tolong sediakan juga file2 yang belum di minify didalam bundle.js karena cukup sulit utk modifikasinya.

thanks

**Comment by meirzadotcom** [comment]
Yep. Insyaa Allah saya cari dulu repositorynya, sudah lama nggak diupdate.

**Comment by rezamukti** [comment]
Ok thx ditunggu kk
